bson-scala/src/test/scala/org/mongodb/scala/bson/BaseSpec.scala

```diff
@@ -15,10 +15,7 @@
  */
 package org.mongodb.scala.bson
 
-import org.junit.runner.RunWith
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
-import org.scalatestplus.junit.JUnitRunner
 
-@RunWith(classOf[JUnitRunner])
```

Comment:
No longer needed thanks to the scala test configurations done in the scala convention.